src/vendors/Datadog.Trace.Vendoring/DatadogLogging.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using Datadog.Trace.Vendoring.Serilog;
+using Datadog.Trace.Vendoring.Serilog.Events;
+using Datadog.Trace.Vendoring.Serilog.Sinks.File;
+
+namespace Datadog.Trace.Vendoring
+{
+    /// <summary>
+    /// Class for Datadog managed file logging.
+    /// </summary>
+    public static class DatadogLogging
+    {
+        private const string WindowsDefaultDirectory = "C:\\ProgramData\\Datadog .NET Tracer\\logs\\";
+        private const string NixDefaultDirectory = "/var/log/datadog/";
+
+        private static readonly long? MaxLogFileSize = 10 * 1024 * 1024;
```

Comment:
Looks like this doesn't need to be nullable, so it can also be `const`.
```suggestion
        private const long MaxLogFileSize = 10 * 1024 * 1024;
```

Reply:
Oops, never mind. I see now that you're setting this in the static ctor.